src/events/touch.js

```diff
@@ -234,6 +234,9 @@ p5.prototype._ontouchend = function(e) {
     if(executeDefault === false) {
       e.preventDefault();
     }
+  } else {
```

Comment:
This change causes problems with iOS (Safari). Specifically, once a sketch is created, it and the entire HTML page it is contained in will be unresponsive to actions that rely on `ontouchend`, such as link-clicking and button pushes. Commenting out the above else-branch makes everything work again.

I recommend that the above code is made conditional for the environment where it is needed (Chrome Mobile 56).

Meanwhile, I'm monkeypatching around the above 'fix'.